recipes/django/meta.yaml

```diff
@@ -0,0 +1,233 @@
+{% set version = "1.8.11" %}
+
+package:
+    name: django
+    version: {{ version }}
+
+source:
+    fn: Django-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/D/Django/Django-{{ version }}.tar.gz
+    md5: 1c85d0d582dc211adc6851dd2dc86228
+    patches:
+        - libgdal.patch
```

Comment:
Indeed that kind of patch we have in `shapely`, `rtree`, and a few others that use `ctypes`. IMO we should always patch conda packages to find the proper conda packaged c-lib that the package is looking for.